src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -188,8 +189,6 @@
 use Symfony\Contracts\Translation\LocaleAwareInterface;
 
 /**
- * FrameworkExtension.
```

Comment:
This comment has been around for over a decade..

Im not against it... Im just saying.